main.go

```diff
@@ -21,7 +21,7 @@ var (
 	sequinsVersion string
 
 	bind       = kingpin.Flag("bind", "Address to bind to. Overrides the config option of the same name.").Short('b').PlaceHolder("ADDRESS").String()
-	root       = kingpin.Flag("root", "Where the sequencefiles are. Overrides the config option of the same name.").Short('r').PlaceHolder("URI").String()
+	source     = kingpin.Flag("source", "Where the sequencefiles are. Overrides the config option of the same name.").Short('r').PlaceHolder("URI").String()
```

Comment:
Since this is just a development branch, I think it's fine. The migration will be a bit tricky, but not too bad; we just `stripe-puppet` before upping sequins again on each node.